### PR TITLE
Replace react-tools for Babel

### PR DESCRIPTION
**Pull request by underbyte**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,4 @@
+.idea/*
```
Comment (gabecase) [inline .gitignore]: JetBrain editor(web storm, rubymine, IntelliJ etc..) settings files ignore in case anyone is wondering

**Comment by willcumbie** [comment]
Seems legit.

**Comment by wpwood** [comment]
Sure

**Comment by adampickeral** [comment]
wait for this https://github.com/rackerlabs/canon-react/pull/21 to go in, and make sure everything still works fine when using the npm module

File: package.json [hunk 2]
```diff
@@ -19,27 +20,29 @@
...
     "karma-phantomjs-launcher": "^0.1.4",
-    "karma-spec-reporter": "0.0.13",
     "karma-react-jsx-preprocessor": "0.1.1",
+    "karma-spec-reporter": "0.0.13",
     "react": "~0.13.2",
-    "reactify": "~1.1.0",
-    "react-tools": "^0.13.2",
-    "karma-browserify": "~3.0.1",
-    "grunt-jsxhint": "~0.5.0"
+    "react-tools": "^0.13.3"
```
Comment (rogeriochaves) [inline package.json]: shouldn't this be removed then?

Comment (underbyte) [inline package.json]: yes

File: README.md [hunk 3]
```diff
@@ -20,7 +20,19 @@ To run tests in chrome and watch for changes run `npm run test-watch`
 `npm install canon-react` will install the canon-react module inside of the `node_modules` directory of your current directory. `npm install canon-react -g` (may need sudo) will install the canon-react module globally.
 
 ### Using Components
-Access canon react components via `var component = require('canon-react').component;` after installation. An example of how to use canon-react components can be found in `node_modules/canon-react/demo/demo.jsx`
+After installation you can use canon rgeact components.
```
Comment (adampickeral) [inline README.md]: typo 'rgeact'

**Comment by adampickeral** [comment]
:+1:

**Pull request merged**